frontend/src/components/StateMultiSelect.js

```diff
@@ -181,7 +181,7 @@ export default function StateMultiSelect({
     <MultiSelect
       name={name}
       control={control}
-      options={stateCodes}
+      options={stateCodes || []}
```

Comment:
The way it was originally coded would lead to infinite loops if the API returned an empty array